WebInstaller/index.html

```diff
@@ -49,10 +45,6 @@
         ></script>
```

Comment:
The PR introduces an esp32webtools directory with what seems like a "local copy" of ESP32 Web Tools. However, the HTML still retrieves the code it uses from unpkg.com. Unless I'm missing something, I'd say that either the script block here should use the local copy, or the local copy (i.e. esp32webtools directory with contents) can be removed?